### PR TITLE
Various governance clarifications

### PR DESCRIPTION
**Pull request by jbeda**
We talked about some of this during the Steering committee meetings.  I don't think any of this should be controversial.

### CODE REVIEW

File: governance.md [hunk 1]
```diff
@@ -144,17 +147,17 @@ to its charter, will own the decision. In the case of extended debate
...
 
-All repositories under Kubernetes github orgs, such as kubernetes and kubernetes-incubator,
-should follow the procedures outlined in the [incubator document](incubator.md). All code projects
-use the [Apache Licence version 2.0](LICENSE). Documentation repositories should use the
-[Creative Commons License version 4.0](https://git.k8s.io/website/LICENSE).
+All new repositories under Kubernetes github orgs should follow the process outlined in the [kubernetes repository guidelines].
+
+All code projects use the [Apache Licence version 2.0](LICENSE). Documentation repositories should use the [Creative Commons License version 4.0](https://git.k8s.io/website/LICENSE).
+
+Note that "Kubernetes incubator" process has been documented in favor of the new guidelines.
```
Comment (dankohn) [inline governance.md]: s/documented/deprecated/

File: committee-steering/governance/README.md [hunk 2]
```diff
@@ -19,6 +19,23 @@ Specific attention has been given to:
 - Process for agreeing upon technical decisions
 - Process for ensuring technical assets remain healthy
 
+## SIG Charter approval process
+
+When introducing a SIG charter or modification of a charter the following process should be used:
+
+- Identify a small set of owners from the SIG to drive the changes.
+  Most typically this'll be the SIG chairs.
```
Comment (smarterclayton) [inline committee-steering/governance/README.md]: nit: `this will`

**Comment by smarterclayton** [comment]
One very minor wording thing, lgtm otherwise.  Will wait for others to read though.

File: governance.md (hunk 1 above)
Comment (jbeda) [reply governance.md]: Good catch. Thanks!

**Comment by jbeda** [comment]
Thanks for the review. Nits/typos fixed.

File: committee-steering/governance/README.md [hunk 3]
```diff
@@ -19,6 +19,23 @@ Specific attention has been given to:
...
 - Process for ensuring technical assets remain healthy
 
+## SIG Charter approval process
+
+When introducing a SIG charter or modification of a charter the following process should be used:
+
+- Identify a small set of owners from the SIG to drive the changes.
+  Most typically this will be the SIG chairs.
+- Work with the steering committee to gain approval.
+  This can simply be submitting a PR and sending mail to [steering@kubernetes.io].
```
Comment (wking) [inline committee-steering/governance/README.md]: Nit: the Markdown spec defines an [implicit link name shortcut][1] (e.g. `[steering@kubernetes.io][]`), but leaving the name brackets off entirely seems to be an extention.  Are we targetting a specific Markdown flavor here?

[1]: https://daringfireball.net/projects/markdown/syntax#link

File: README.md [hunk 4]
```diff
@@ -13,16 +13,26 @@ issues, mailing lists, conferences, etc.
...
 
-## SIGs
+## Governance
 
-Kubernetes is a set of subprojects, each shepherded by a Special Interest Group (SIG).
+Kubernetes has three types of groups that are officially supported:
 
-A first step to contributing is to pick from the [list of kubernetes SIGs](sig-list.md).
+* **Committees** are named sets of people that are chartered to take on sensitive topics.
+  This group is encouraged to be as open as possible while achieving its mission.
```
Comment (cblecker) [inline README.md]: Is it worth mentioning that while they are encouraged to be open, by the nature of the topics they discuss committees are permitted to have private communications as well?

File: README.md [hunk 5]
```diff
@@ -13,16 +13,26 @@ issues, mailing lists, conferences, etc.
...
-A SIG can have its own policy for contribution,
-described in a `README` or `CONTRIBUTING` file in the SIG
-folder in this repo (e.g. [sig-cli/CONTRIBUTING](sig-cli/CONTRIBUTING.md)),
-and its own mailing list, slack channel, etc.
+See the [full governance doc](governance.md) for more details on these groups.
+
+Kubernetes is a set of subprojects, each shepherded by a Special Interest Group (SIG).
+Most decision making in Kubernetes happens through SIGs.
+
+A SIG can have its own policy for contribution, described in a `README` or `CONTRIBUTING` file in the SIG folder in this repo (e.g. [sig-cli/CONTRIBUTING](sig-cli/CONTRIBUTING.md)), and its own mailing list, slack channel, etc.
```
Comment (cblecker) [inline README.md]: The link goes to CONTRIBUTING.md but the text only says CONTRIBUTING.

File: committee-steering/governance/README.md [hunk 6]
```diff
@@ -19,6 +19,23 @@ Specific attention has been given to:
 - Process for agreeing upon technical decisions
 - Process for ensuring technical assets remain healthy
 
+## SIG Charter approval process
+
+When introducing a SIG charter or modification of a charter the following process should be used:
+
+- Identify a small set of owners from the SIG to drive the changes.
+  Most typically this will be the SIG chairs.
+- Work with the steering committee to gain approval.
```
Comment (cblecker) [inline committee-steering/governance/README.md]: Wouldn't working with the steering to gain approval be a latter step in the process (after circulating changes with the sig and/or community)?

File: committee-steering/governance/README.md (hunk 3 above)
Comment (cblecker) [reply committee-steering/governance/README.md]: @wking links are at the bottom of the doc (this line is on line 62)

File: governance.md [hunk 7]
```diff
@@ -109,6 +110,8 @@ This is the purpose of Working Groups (WG). The intent is to make
 Working Groups relatively easy to create and to deprecate, once
 inactive.
 
+Working groups do not own any code or subprojects. Instead, they are a place for people to discuss topics that cross SIG boundaries.
```
Comment (cblecker) [inline governance.md]: nit: the rest of this doc is wrapped at 80 chars

File: governance.md [hunk 8]
```diff
@@ -144,17 +147,17 @@ to its charter, will own the decision. In the case of extended debate
 or deadlock, decisions may be escalated to the Steering Committee,
 which is expected to be uncommon.
 
-The exact processes and guidelines for such cross-project
-communication have yet to be formalized, but when in doubt, use
-kubernetes-dev@googlegroups.com and make an announcement at the
-community meeting.
+The [KEP process] is being developed as a way to facilitate definition, agreement and communication of efforts that cross SIG boundaries.
```
Comment (cblecker) [inline governance.md]: nit: the rest of this doc is wrapped at 80 chars

File: committee-steering/governance/README.md (hunk 3 above)
Comment (wking) [reply committee-steering/governance/README.md]: > links are at the bottom of the doc (this line is on line 62)

Right, and those are fine.  But the syntax used in *this* line (withought the trailng `[]`) isn't in the spec.  It's the "completely omit" case from the [Kramdown spec][1], but I don't know how portable it is to other Markdown engines.

[1]: https://kramdown.gettalong.org/syntax.html#reference-links

File: README.md (hunk 4 above)
Comment (wking) [reply README.md]: > Is it worth mentioning that while they are encouraged to be open, by the nature of the topics they discuss committees are permitted to have private communications as well?

I think "as possible" already suggests this, but more explicit wording would be fine too.

**Comment by bgrant0607** [comment]
See also https://github.com/kubernetes/community/pull/1986

File: README.md [hunk 9]
```diff
@@ -13,16 +13,26 @@ issues, mailing lists, conferences, etc.
...
+## Governance
 
-Kubernetes is a set of subprojects, each shepherded by a Special Interest Group (SIG).
+Kubernetes has three types of groups that are officially supported:
 
-A first step to contributing is to pick from the [list of kubernetes SIGs](sig-list.md).
+* **Committees** are named sets of people that are chartered to take on sensitive topics.
+  This group is encouraged to be as open as possible while achieving its mission.
+  Examples of committees include the steering committee and things like security or code of conduct.
+* **Special Interest Groups (SIGs)** are persistent open groups that focus on a part of the project.
```
Comment (pwittrock) [inline README.md]: what does it mean to be open in membership?

Comment (parispittman) [reply README.md]: transparent?

File: committee-steering/governance/README.md (hunk 3 above)
Comment (pwittrock) [reply committee-steering/governance/README.md]: I think this is fine.  We do it elsewhere in the repo.

File: committee-steering/governance/README.md [hunk 10]
```diff
@@ -19,6 +19,23 @@ Specific attention has been given to:
...
+  This can simply be submitting a PR and sending mail to [steering@kubernetes.io].
+  If more substantial changes are desired it is advisable to socialize those before drafting a PR.
+    - The steering committee will be looking to ensure the scope is reasonable (and within the scope of Kubernetes) and that processes are fair.
+- Alert the rest of the SIG in question to the changes so that anyone within the SIG can object.  
+  Including the SIG mailing list in communications with the steering committee would work for this.
+- For large changes alert the rest of the Kubernetes community as the scope of the changes becomes clear.
```
Comment (pwittrock) [inline committee-steering/governance/README.md]: May also want to announce at the community meeting

File: README.md (hunk 9 above)
Comment (wking) [reply README.md]: > > what does it mean to be open in membership?
>
> transparent?

An alternative interpretation would be that that self-nomination is sufficient to join (i.e. that there's no new-member vetting process).  Perhaps the wording here (and/or in `sig-governance.md`?) could be updated to make it clear which of those is intended?  Do non-lead SIG members have any explicit powers?

File: committee-steering/governance/README.md [hunk 11]
```diff
@@ -19,6 +19,23 @@ Specific attention has been given to:
 - Process for agreeing upon technical decisions
 - Process for ensuring technical assets remain healthy
 
+## SIG Charter approval process
+
+When introducing a SIG charter or modification of a charter the following process should be used:
+
+- Identify a small set of owners from the SIG to drive the changes.
+  Most typically this will be the SIG chairs.
```
Comment (pwittrock) [inline committee-steering/governance/README.md]: What is the process for a new SIG?  Presumably it folks proposing the SIG agree upon the chairs / tech leads?

File: committee-steering/governance/README.md [hunk 12]
```diff
@@ -19,6 +19,23 @@ Specific attention has been given to:
...
+- Work with the steering committee to gain approval.
+  This can simply be submitting a PR and sending mail to [steering@kubernetes.io].
+  If more substantial changes are desired it is advisable to socialize those before drafting a PR.
```
Comment (pwittrock) [inline committee-steering/governance/README.md]: I'd expect the SIG to have consensus before asking the SC to review the changes.

Comment (wking) [reply committee-steering/governance/README.md]: > I'd expect the SIG to have consensus before asking the SC to review the changes.

Or to have deadlocked?

File: committee-steering/governance/README.md (hunk 3 above)
Comment (jbeda) [reply committee-steering/governance/README.md]: This is already used in this file and I was just following the pattern.  If and when we start using the content with a markdown parser that doesn't respect this syntax we can look to fix it. As it is, this is part of the commonmark spec which github users (https://spec.commonmark.org/0.28/#shortcut-reference-link).  But the state of "spec" in the markdown world is a disaster and I really don't want to have that argument.

File: committee-steering/governance/README.md (hunk 6 above)
Comment (jbeda) [reply committee-steering/governance/README.md]: I was structuring this as an OARP type thing -- Owners, Approvers, Reviewers and Participants.  I'll rework it so that it is viewed as more of a sequential process.

File: README.md (hunk 9 above)
Comment (jbeda) [reply README.md]: Right now in the steering committee we are working on guidance/process for the level of "membership" needed to participate in processes like voting for the steering committee.  As part of that I don't want to overload the term "membership".  I'll reword this to say that anyone is welcome to contribute and participate.

File: committee-steering/governance/README.md (hunk 11 above)
Comment (jbeda) [reply committee-steering/governance/README.md]: Unfortunately, this process is split between this doc and `/sig-governance.md`.  We should probably rationalize these but that is a bigger change.  Advice would be appreciated.

I *did* change that in this PR too to say that for creating a SIG a set of folks proposing the SIG should work with the Steering Committee to scope it.

File: committee-steering/governance/README.md (hunk 12 above)
Comment (jbeda) [reply committee-steering/governance/README.md]: Reordered to make it more clear. (pushing soon).

File: README.md [hunk 13]
```diff
@@ -13,16 +13,27 @@ issues, mailing lists, conferences, etc.
 
 For more specific topics, try a SIG.
 
-## SIGs
+## Governance
 
-Kubernetes is a set of subprojects, each shepherded by a Special Interest Group (SIG).
+Kubernetes has three types of groups that are officially supported:
```
Comment (bgrant0607) [inline README.md]: This omits subprojects.

https://github.com/kubernetes/community/blob/master/governance.md#community-groups

Comment (jbeda) [reply README.md]: So -- I'm torn on this and would love advice.  In my mind subprojects are a subunit of a SIG.  They are scoped and live/die and are governed by the owning SIG.  It is up to the SIG to decide how tightly those sub-projects are controlled and what the process for figuring out who has commit permissions on those.

I'm happy to rationalize this and *either* demote them to be sub-SIG in governance.md or add them here.  Let me know what you prefer.

Comment (cblecker) [reply README.md]: Opinion: because they are created and destroyed by the sigs, they should be at a sub-sig level.

Comment (bgrant0607) [reply README.md]: Yes, but lack of awareness of subprojects leads to attempts to form WGs that should really be subprojects.

I would be fine with a mention under the SIG bullet rather than just below.

File: README.md [hunk 14]
```diff
@@ -13,16 +13,27 @@ issues, mailing lists, conferences, etc.
...
 
-A first step to contributing is to pick from the [list of kubernetes SIGs](sig-list.md).
+* **Committees** are named sets of people that are chartered to take on sensitive topics.
+  This group is encouraged to be as open as possible while achieving its mission but, because of the nature of the topics discussed, private communications are allowed.
+  Examples of committees include the steering committee and things like security or code of conduct.
+* **Special Interest Groups (SIGs)** are persistent open groups that focus on a part of the project.
+  SIGs must have open and transparent proceedings.
+  Anyone is welcome to participate and contribute provided they follow the Kubernetes Code of Conduct.
+  The purpose of a SIG is to own and develop a set of subprojects (defined below).
```
Comment (bgrant0607) [inline README.md]: Please bold "subprojects"

File: committee-steering/governance/README.md [hunk 15]
```diff
@@ -19,6 +19,25 @@ Specific attention has been given to:
...
+
+- Identify a small set of Owners from the SIG to drive the changes.
+  Most typically this will be the SIG chairs.
+- Work with the rest of the SIG in question (Reviewers) to craft the changes.
+  Make sure the keep the SIG in the loop as discussions progress with the Steering Committee (next step).
+  Including the SIG mailing list in communications with the steering committee would work for this.
+- Work with the steering committee (Approvers) to gain approval.
+  This can simply be submitting a PR and sending mail to [steering@kubernetes.io].
+  If more substantial changes are desired it is advisable to socialize those before drafting a PR.
+    - The steering committee will be looking to ensure the scope is reasonable (and within the scope of Kubernetes) and that processes are fair.
```
Comment (bgrant0607) [inline committee-steering/governance/README.md]: This is the SIG's scope or the charter's scope?

File: committee-steering/governance/README.md [hunk 16]
```diff
@@ -19,6 +19,25 @@ Specific attention has been given to:
 - Process for agreeing upon technical decisions
 - Process for ensuring technical assets remain healthy
 
+## SIG Charter approval process
+
+When introducing a SIG charter or modification of a charter the following process should be used.  
+As part of this we will define roles for the OARP process (Owners, Approvers, Reviewers, Participants)
```
Comment (bgrant0607) [inline committee-steering/governance/README.md]: Worth linking to an explanation of OARP?

Or, they could just search, I suppose. The top hit is reasonable?
https://stumblingabout.com/tag/oarp/

File: governance.md [hunk 17]
```diff
@@ -144,17 +147,17 @@ to its charter, will own the decision. In the case of extended debate
...
 
 # Repository guidelines
 
-All repositories under Kubernetes github orgs, such as kubernetes and kubernetes-incubator,
-should follow the procedures outlined in the [incubator document](incubator.md). All code projects
-use the [Apache Licence version 2.0](LICENSE). Documentation repositories should use the
-[Creative Commons License version 4.0](https://git.k8s.io/website/LICENSE).
+All new repositories under Kubernetes github orgs should follow the process outlined in the [kubernetes repository guidelines].
+
+All code projects use the [Apache Licence version 2.0](LICENSE). Documentation repositories should use the [Creative Commons License version 4.0](https://git.k8s.io/website/LICENSE).
```
Comment (bgrant0607) [inline governance.md]: Should we move this to the kubernetes-repositories doc?

There's probably other content from the incubator doc that should be moved there, also.

File: committee-steering/governance/README.md (hunk 16 above)
Comment (mattfarina) [reply committee-steering/governance/README.md]: This this is governance related I think it's worth being explicit and not leaving it up to the reader to work it out. A direct link would be useful. Just my 2 cents.

File: governance.md [hunk 18]
```diff
@@ -55,8 +55,8 @@ itself. Examples:
 * Horizontal: Scalability, Architecture
 * Project: Testing, Release, Docs, PM, Contributor Experience
 
-SIGs must have at least one and ideally two SIG leads at any given
-time. SIG leads are intended to be organizers and facilitators,
+SIGs must have at least one and ideally two SIG chairs at any given
+time. SIG chairs are intended to be organizers and facilitators,
```
Comment (mattfarina) [inline governance.md]: The only SIG with multiple chairs/leads from the same company is SIG VMWare. The rest, with multiple chairs, are from differing companies (see the [sig list](https://github.com/kubernetes/community/blob/master/sig-list.md)). When it comes to [community membership](https://github.com/kubernetes/community/blob/master/community-membership.md) we have a sponsor requirement of:

> Sponsors must be from multiple member companies to demonstrate integration across community.

For SIG Chairs do we want something similar?

Comment (cblecker) [reply governance.md]: @mattfarina I don't think it's a problem unless the *sig* feels it's a problem.

File: governance.md (hunk 7 above)
Comment (dlorenc) [inline governance.md]: Do you think it would be worth adding a clarification that you do not need a working group to collaborate with others in the community? This seems to be a common theme in the various WG/SIG proposal discussions going on right now.

File: governance.md (hunk 18 above)
Comment (jbeda) [reply governance.md]: I'm going to leave this to another update to keep this PR from sprawling.  I was just fixing s/leads/chairs/ here.  There are concerns here around how we curate voting members for the steering committee that we are actively working through now.  This is part of that hairball.

File: governance.md (hunk 7 above)
Comment (jbeda) [reply governance.md]: Good idea -- added in the next revision.

**Comment by jbeda** [comment]
@bgrant0607 PTAL -- I *think* this gets us to the right place wrt positioning subprojects.

Note that I'm going to be on vacation next week so if we don't get this done by then there'll be some latency.

**Comment by pwittrock** [comment]
This LGTM.  I expect us to continue to iterate and refine all of the governance docs for some time.

File: committee-steering/governance/README.md [hunk 19]
```diff
@@ -19,6 +19,25 @@ Specific attention has been given to:
...
 
+## SIG Charter approval process
+
+When introducing a SIG charter or modification of a charter the following process should be used.
+As part of this we will define roles for the [OARP] process (Owners, Approvers, Reviewers, Participants)
+
+- Identify a small set of Owners from the SIG to drive the changes.
+  Most typically this will be the SIG chairs.
+- Work with the rest of the SIG in question (Reviewers) to craft the changes.
+  Make sure the keep the SIG in the loop as discussions progress with the Steering Committee (next step).
```
Comment (derekwaynecarr) [inline committee-steering/governance/README.md]: nit: make sure to keep

File: governance.md [hunk 20]
```diff
@@ -109,6 +110,12 @@ This is the purpose of Working Groups (WG). The intent is to make
 Working Groups relatively easy to create and to deprecate, once
 inactive.
 
+Working groups do not own any code or subprojects. Instead, they are a place for people to discuss topics that cross SIG boundaries.
+
+Working groups are primarily used to facilitate topics of discussion that are in scope for Kubernetes but that cross SIG lines.
```
Comment (derekwaynecarr) [inline governance.md]: not sure where to state this, but like a SIG, WG meetings should default to public, be recorded for later viewing, etc.

Comment (jbeda) [reply governance.md]: Hmm -- I'll add a note but we should make sure this is handled as part of #1997. SIG stuff is in sig-governance.md.

**Comment by philips** [comment]
LGTM

**Comment by idvoretskyi** [comment]
LGTM from me.

File: governance.md (hunk 18 above)
Comment (cantbewong) [reply governance.md]: Regarding the vmware SIG, the intent is that we will actively seek a non vmware employee who is willing and able to assist in SIG leadership roles and responsibilities. The goal is to ensure that this SIG is open to all interested parties. The initial leadership composition is a short term artifact of bootstrapping the SIG. Until, and even after diversity is achieved, the SIG intends to actively engage in recruitment to include users, partners and any others interested.

File: governance.md [hunk 21]
```diff
@@ -109,6 +110,16 @@ This is the purpose of Working Groups (WG). The intent is to make
...
+If a set of folks in the community want to get together and discuss a topic, they can do so without forming a Working Group.
```
Comment (bgrant0607) [inline governance.md]: It seems like this paragraph and the first paragraph should be merged or refactored. 

I also want to clarify the "short-lived" aspect. We expect WGs to not live forever -- they should have a goal and concrete deliverables, but being short-lived shouldn't be a reason to create a WG. For example, a WG isn't needed for every short-term effort. Every release ships the results of many short-term efforts.

I also do want to address the class of SIGs/WGs for "use case X on Kubernetes": most of SIG Apps, SIG Big Data, M L WG, the proposed IoT/Edge WG. I will send email about that.

Most of our current WGs probably should not be WGs.

**Comment by philips** [comment]
@brendanburns and I had to answer questions regarding this stuff. Particularly the commitee, sig, and subproject thing. Can you rebase and merge this @jbeda ?

**Comment by michelleN** [comment]
@jbeda - this is a really helpful document. would you mind re-basing so we can merge?

**Comment by jdumars** [comment]
Thank you Joe for getting this done!

File: governance.md [hunk 22]
```diff
@@ -106,15 +107,28 @@ This is the purpose of Working Groups (WG). The intent is to make
 Working Groups relatively easy to create and to deprecate, once
 inactive.
 
-To propose a new working group, first find a SIG to sponsor the group. 
+Working groups do not own any code or subprojects. Instead, they are a place for
```
Comment (philips) [inline governance.md]: we kept getting hung up on this during the WG discussion. I think everyone agreed working groups can create and prototype. Just that the code landing in Kubernetes is contingent on SIG/subproject approval. "own any code" is confusing in this meaning.

Comment (quinton-hoole-2) [reply governance.md]: Agreed.  I think that this needs to be addressed further up in the doc too.

File: README.md [hunk 23]
```diff
@@ -13,16 +13,27 @@ issues, mailing lists, conferences, etc.
...
+  Examples of committees include the steering committee and things like security or code of conduct.
+* **Special Interest Groups (SIGs)** are persistent open groups that focus on a part of the project.
+  SIGs must have open and transparent proceedings.
+  Anyone is welcome to participate and contribute provided they follow the Kubernetes Code of Conduct.
+  The purpose of a SIG is to own and develop a set of **subprojects**.
+  * **Subprojects** Each SIG can have a set of subprojects.
+    These are smaller groups that can work independently.
+    Some subprojects will be part of the main Kubernetes deliverables while others will be more speculative and live in the `kubernetes-sigs` github org.
+* **Working Groups** are temporary groups that are formed to address issues that cross SIG boundaries.
+  Working groups do not own any code or other long term artifacts.
```
Comment (quinton-hoole-2) [inline README.md]: See @philips comment below.  Clarify to make it clear that prototypes and proof-of-concept implementations by working groups are in scope and encouraged.

File: governance.md (hunk 18 above)
Comment (quinton-hoole-2) [reply governance.md]: I think that having chairs from multiple companies/organizations should be very strongly encouraged, and that single-company groups of chairs should be the exception, and clearly justified when this is required for practical reasons.

File: governance.md [hunk 24]
```diff
@@ -106,15 +107,28 @@ This is the purpose of Working Groups (WG). The intent is to make
...
+people to discuss topics that cross SIG boundaries.
+
+Working groups are primarily used to facilitate topics of discussion that are in
+scope for Kubernetes but that cross SIG lines. If a set of folks in the
+community want to get together and discuss a topic, they can do so without
+forming a Working Group. As a community we will be looking for other ways to
+highlight and encourage a larger ecosystem (with things like slack channels)
+without offering any official endorsement.
+
+To propose a new working group, first find a SIG to sponsor the group.
```
Comment (quinton-hoole-2) [inline governance.md]: Given that WG's are intended to cross SIGs, I think that multiple sigs should sponsor.  If only one SIG is in support of the WG, it can simply live in the SIG with all other discussions and/or sub-projects there.

File: governance.md (hunk 22 above)
Comment (smarterclayton) [reply governance.md]: How about:

“Since WG are intended to have a definite lifetime, to prevent orphaned code at the dissolution of a WG all code in Kubernetes must have a SIG to own and maintain it.  The sponsoring SIGs for a WG are expected to own and manage the code created for Kubernetes by a WG, and have authority on whether code created in a WG should become part of Kubernetes”

Or similar?

Comment (quinton-hoole-2) [reply governance.md]: Perhaps a bit wordy, but otherwise fine.  It can be word-smithed in a followup PR, IMO.

File: README.md (hunk 23 above)
Comment (bgrant0607) [reply README.md]: We have a repository policy:
https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md

This should not change it.

File: governance.md (hunk 22 above)
Comment (bgrant0607) [reply governance.md]: @philips @smarterclayton @quinton-hoole 

We have a repository policy:
https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md

This PR should not change it.

**Comment by brendanburns** [comment]
Agree, we should just point at the repo policy here.

"Like any other code related to kubernetes, any code created by a WG will
be governed by the repo policy described here."

**Comment by philips** [comment]
Filed https://github.com/kubernetes/community/issues/2766 in an effort to close out this 7 month old PR. 

Thanks @jbeda and reviewers.
